### PR TITLE
Add setup instructions to readme

### LINKED ISSUE #12: Add documentation for BE repo local deployment to Readme

### PR DESCRIPTION
**Pull request by msimon42**
### Sidebar Checklist
​
- [ ] Ran test suite
- [ ] Request reviewers
- [ ] Assign yourself and other contributors
- [ ] Link to project
- [x ] Added to Readme
​
### Issues Resolved
Resolves #12
​
### Problem Addressed
​
### Solution Implemented
​
### Screenshots (optional)
​
### Other Notes
Adds setup instructions to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
...
+
+3. Set environment variables:
+```
+$ export FLASK_APP=flaskr/app
+$ export FLASK_ENV=development
+```
+4. Migrate database:
+```
+$ python flaskr/manage.py db init
+$ python flaskr/manage.py db migrate
```
Comment (DanielEFrampton) [inline README.md:20]: These first two aren't necessary for running the migrations, they're part of the setup process; see my version of the README, we'll need to merge ours together

File: README.md [hunk 2]
```diff
@@ -1 +1,25 @@
 # rosetta-be
+## Local Setup Instructions
+
+1. Set up virtual environment:
+```
+$ python3 -m venv venv
+$ . venv/bin/activate
```
Comment (DanielEFrampton) [inline README.md:7]: I never took this step; is it too late for me to setup a virtual environment?

File: README.md [hunk 3]
```diff
@@ -1 +1,25 @@
...
+```
+$ export FLASK_APP=flaskr/app
+$ export FLASK_ENV=development
+```
+4. Migrate database:
+```
+$ python flaskr/manage.py db init
+$ python flaskr/manage.py db migrate
+$ python flaskr/manage.py db upgrade
+```
```
Comment (mintona) [inline README.md:22]: Which of these are one time commands vs. having to run for each new master update? (I'm guessing install -r requirements is like bundle and migrate after migrations but what about the env vars?)

**Comment by DanielEFrampton** [comment]
@msimon42 please fill out the PR template!

**Pull request merged**